workflow/rules/maftools.smk

```diff
@@ -0,0 +1,36 @@
+rule group_bcf_to_vcf:
+    input:
+        "results/final-calls/{group}.{event}.{calling_type}.fdr-controlled.bcf",
+    output:
+        temp("results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf"),
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    params:
+        extra="",
+    wrapper:
+        "v3.8.0/bio/bcftools/view"
+
+
+rule group_vcf_to_maf:
+    input:
+        vcf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf",
+        ref=genome,
+    output:
+        maf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.maf",
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    conda:
+        "../envs/vcf2maf.yaml"
+    params:
+        genome_build=config["ref"]["build"],
+        species=config["ref"]["species"],
+    shell:
+        "vcf2maf.pl --inhibit-vep "
+        " --retain-info PROB_ARTIFACT,PROB_ABSENT,PROB_SOMATIC,PROB_GERMLINE_HOM,PROB_GERMLINE_HET "
+        " --retain-fmt DP,AF "
+        " --ncbi-build {params.genome_build} "
+        " --species {params.species} "
+        " --vcf-tumor-id tumor --vcf-normal-id panel_of_normals "
```

Comment:
Is maf always tumor/normal? Then the config should contain a setting for specifying the proper alias to use for both.

Reply:
Normal is optional, and this is now handled properly via the config.